app/src/main/java/fr/free/nrw/commons/delete/DeleteHelper.java

```diff
@@ -59,7 +59,8 @@ public DeleteHelper(MediaWikiApi mwApi,
      * @return
      */
     public Single<Boolean> makeDeletion(Context context, Media media, String reason) {
-        viewUtil.showShortToast(context, "Trying to nominate " + media.getDisplayTitle() + " for deletion");
+        viewUtil.showShortToast(context, context.getString(R.string.delete_helper_make_deletion_toast_1)
```

Comment:
Please use only one string here, and get media.getDisplayTitle() as a variable into it. For more info see my comment at strings.xml file.